l10n_it_fiscal_document_type/__manifest__.py

```diff
@@ -12,7 +13,10 @@
     'website': 'https://github.com/OCA/l10n-italy'
                '/tree/12.0/l10n_it_fiscal_document_type',
     'license': 'AGPL-3',
-    'depends': ['l10n_it_account'],
+    'depends': [
+        'l10n_it_account',
+        'mail',
```

Comment:
come mai vuoi aggiungere questa dipendenza?

Reply:
Perché nel test uso `mail_new_test_user` che pensavo venisse dai test del modulo **mail**, ora però vedo che viene dal modulo **test_mail** che in effetti contiene proprio i test per il modulo **mail**; quindi mettere uno o l'altro potrebbe essere equivalente.
Se preferisci metto **test_mail** invece di **mail**.

Reply:
> Se preferisci metto **test_mail** invece di **mail**.

sì preferirei grazie

Reply:
> > Se preferisci metto **test_mail** invece di **mail**.
> 
> sì preferirei grazie

Fatto in https://github.com/OCA/l10n-italy/pull/3535